app/0.14.x/plugin-development/plugin-configuration.md

```diff
@@ -76,6 +79,9 @@ return {
   end
 }
 ```
+[Back to TOC](#table-of-contents)
```

Comment:
Fenced code blocks should have a blank line before and after - please check your various additions and add blank lines as necessary, thanks.

Reply:
https://help.github.com/articles/creating-and-highlighting-code-blocks/#fenced-code-blocks